CHANGELOG.md

```diff
@@ -3,6 +3,133 @@ All notable changes to this project will be documented in this file.
+ - JSON-RPC: `showrunes` new field `last_used` ([#6617])
+ - New configurable Content-Security-Policy (CSP) header for clnrest ([#6686])
+ - New configurable Cross-Origin-Resource-Sharing(CSP) header for clnrest ([#6686])
+
+
+### Changed
+
+ - Spec: dual-funding now follows the next-funding-id rules. ([#6824])
+ - RPC `listpeerchannels`.`inflights` may sometimes not include `scratch_txid` (mandatory -> optional) ([#6824])
+ - RPC: `openchannel_update` will now echo back a result if there's a matching inflight record for this open. ([#6824])
+ - RPC: `openchannel_signed` will now remember the details of a signed PSBT even if the peer is disconnected. ([#6824])
+ - Config: `large-channels` is now the default, wumbology for all. ([#6783])
+ - cln-plugin: Suppress internal logging handler via `with_logging(false)` ([#6797])
+ - JSON-RPC: `checkrune` `rate` restriction is slightly stricter (exact division of time like `per`) ([#6710])
+ - Upgraded clnrest to poetry project. ([#6651])
+ - JSON-RPC time fields now have full nanosecond precision (i.e. 9 decimals not 3): `listfowards` `received_time` `resolved_time` `listpays`/`listsendpays` `created_at`. ([#6617])
+ - Protocol: `invoice` no longer explicitly encodes `c` if it's the default (18) ([#6668])
+ - Protocol: we no longer disconnect every time we receive a warning message. ([#6668])
+
+
+### Deprecated
+
+Note: You should always set `allow-deprecated-apis=false` to test for changes.
+
+
+
+### Removed
+
+ - build: `--enable-developer` arg to configure (and DEVELOPER variables): use `./configure --enable-debugbuild` and `developer` setting at runtime. ([#6311])
+ - `dev-sendcustommsg` (use `sendcustommsg`, which was added in v0.10.1) ([#6311])
+
+
+### Fixed
+
+ - Some peer disconnects due to update_fee disagreements are avoided. ([#6833])
+ - websocket server notifications are available with ([#6749])
+ - Issue splicing with pending / stuck HTLCs fixed. ([#6748])
+ - wallet: close change outputs show up immediately in `listfunds` so you can CPFP. ([#6734])
+ - Restore any missing metadata that resource constrained signers stripped ([#6767])
+ - fix `checkrune` when `method` parameter is the empty string. ([#6759])
+ - JSON-RPC: `getroute` now documents that it ignores `fuzzpercent`. ([#6697])
+ - rune: use runes table `id` instead `runes_uniqueid` from `vars` because it returns incorrect unique id if rune/s migrated from datastore. ([#6715])
+ - Added docs, testing, and some fixes related to splicing out, insufficent balance handling, and restarting during a splice. ([#6677])
+
+
+### EXPERIMENTAL
+
+ - Fixed anchor spending to be able to use more than one UTXO. ([#6780])
+ - JSON-RPC: added new dual-funding state `DUALOPEND_OPEN_COMMITTED` ([#6628])
+
+
+[#6628]: https://github.com/ElementsProject/lightning/pull/6628
+[#6753]: https://github.com/ElementsProject/lightning/pull/6753
+[#6753]: https://github.com/ElementsProject/lightning/pull/6753
```

Comment:
#6753 is linked 6 times!  I think deduplicating the links would keep the changelog more compact.